portability-core/src/main/java/org/dataportabilityproject/cloud/local/LocalCloudFactory.java

```diff
@@ -46,21 +61,35 @@ public JobDataCache load(String service) throws Exception {
         }
       });
 
-  private static final Supplier<PersistentKeyValueStore> KEY_VALUE_SUPPLIER =
-      Suppliers.memoize(InMemoryPersistentKeyValueStore::new);
+  // Google DataStore emulator is used for local development
```

Comment:
Nice find on the local data store! Does the code here check if it's running and print an error if not?

Reply:
The database client code gives an error on startup if one isn't running.